src/Compiler/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntermediateNodeWriter.cs

```diff
@@ -336,11 +335,11 @@ protected void WriteDiagnostics(IntermediateNode node)
 
                 // Purposefully not writing out the entire message to ensure readable IR and because messages
                 // can span multiple lines. Not using string.GetHashCode because we can't have any collisions.
-                using (var md5 = MD5.Create())
+                using (var sha256 = HashAlgorithmOperations.Create())
```

Comment:
Nit: This variable name should probably change to just `hashAlgorithm`, since it might not be sha256 any more